<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelOpenai">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ChatGPT Provider&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditTemperature">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>66</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="plainText">
     <string>0.7</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditApikey">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>101</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemperature">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>71</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Temperature&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelApikey">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>API Key</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderTemperature">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelConfiguration">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Configuration&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelModel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditModel">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>142</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelEndpoint">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Endpoint</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditEndpoint">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>183</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioOpenai">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>OpenAI</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioAzureopenai">
        <property name="text">
         <string>Azure OpenAI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditFileoutput">
    <property name="geometry">
     <rect>
      <x>561</x>
      <y>250</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileoutput">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>File Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFormatoutput">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Format</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>239</y>
      <width>811</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEditFileinput">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>256</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>31</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="labelFileinput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>File Input</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelColumns">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Columns</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSeparatorinput">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSeperatorinput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Separator</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSelectColumns">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>121</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Columns</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFormatinput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxFormatinput">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>50</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>csv</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelSeperatoroutput">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Separator</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSeparatoroutput">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxFormatoutput">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>54</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>csv</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEditSampleoutputsize">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSampleinputsize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>No. of samples</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSampleinputsize">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSamplesize">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>121</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>No. of samples</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonPreview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGenerate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>811</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>labelOpenai</zorder>
   <zorder>plainTextEditTemperature</zorder>
   <zorder>plainTextEditApikey</zorder>
   <zorder>labelTemperature</zorder>
   <zorder>labelApikey</zorder>
   <zorder>horizontalSliderTemperature</zorder>
   <zorder>labelConfiguration</zorder>
   <zorder>labelModel</zorder>
   <zorder>plainTextEditModel</zorder>
   <zorder>labelEndpoint</zorder>
   <zorder>plainTextEditEndpoint</zorder>
   <zorder>plainTextEditFileoutput</zorder>
   <zorder>labelFileoutput</zorder>
   <zorder>labelFormatoutput</zorder>
   <zorder>pushButtonPreview</zorder>
   <zorder>pushButtonGenerate</zorder>
   <zorder>tableView</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
